src/routes/github/create-branch/github-create-branch-post.ts

```diff
@@ -40,6 +40,20 @@ export const GithubCreateBranchPost = async (req: Request, res: Response): Promi
 		res.sendStatus(200);
 	} catch (err) {
 		req.log.error({ err }, "Error creating branch");
-		res.sendStatus(500);
+		res.status(500).json(verifyError(err));
 	}
 };
+
+const verifyError = (error) => {
+	switch (error.status) {
+		case 403:
+			// TODO: Fix the url later, once you figure out how to get the `installationId`
+			return ["This GitHub repository hasn't been configured to your Jira site. <a href='#'>Allow access to this repository.</a>"];
```

Comment:
Are these returning as an array because the error hbs file expect this?

Reply:
Yep. The error messages from GitHub is super vague and we can't be fully sure that the error messages that we have are the only reasons for causing that error. For example, 403 may be caused by other reasons too(which I haven't seen yet) and not just the one we have, thats why instead of using a normal string, changed that to an array to show the list of possible reasons for the failure.